bin/node-template/node/src/service.rs

```diff
@@ -111,6 +112,7 @@ pub fn new_full(config: Configuration) -> Result<TaskManager, ServiceError> {
 
 	let role = config.role.clone();
 	let force_authoring = config.force_authoring;
+	let backoff_authoring_blocks: Option<BackoffAuthoringOnFinalizedHeadLagging<_>> = None;
```

Comment:
Maybe we should implement the trait for `()` to replace this code here by:
```suggestion
	let backoff_authoring_blocks: Option<()> = None;
```

Reply:
That's a good idea!